Device/_Template_Flash/FlashPrg.c

```diff
@@ -50,18 +52,22 @@
 */
 
 
+
 /*
  *  Initialize Flash Programming Functions
  *    Parameter:      adr:  Device Base Address
  *                    clk:  Clock Frequency (Hz)
  *                    fnc:  Function Code (1 - Erase, 2 - Program, 3 - Verify)
  *    Return Value:   0 - OK,  1 - Failed
  */
-
 int Init (unsigned long adr, unsigned long clk, unsigned long fnc) {
 
   /* Add your Code */
-  return (0);                                  // Finished without Errors
+  UNUSED(adr);
+  UNUSED(clk);
+  UNUSED(fnc);
+  
+  return (0);                                            /* Finished without Errors */
```

Comment:
Why do we keep the return value in brackets?

Reply:
I put brackets around the return values to make it more readable but I will remove the brackets.

Reply:
Well, we have this in many places. We can keep it if you think its more readable.